src/mlpack/methods/dbscan/random_point_selection.hpp

```diff
@@ -27,16 +27,24 @@ class RandomPointSelection
   /**
    * Select the next point to use, randomly.
    *
-   * @param unvisited Bitset indicating which points are unvisited.
-   * @param data Unused data.
+   * @param point Unused data.
+   * @param data Dataset to cluster.
    */
   template<typename MatType>
-  static size_t Select(const boost::dynamic_bitset<>& unvisited,
-                       const MatType& /* data */)
+  size_t Select(const size_t /* point */,
+                       const MatType& data /* data */)
   {
-    const size_t max = unvisited.count();
+    // Initialize the length of the unvisited bitset
+    size_t size = data.n_cols; // Get the size of points.
+    if (unvisited.size() != size) 
+    {
+      unvisited.resize(size, true); // Resize & Set bitset to one
```

Comment:
Missing point at the end of the comment, line 37 as well.

Reply:
Thank you for pointing that out!!